client/state/current-user/actions.js

```diff
@@ -19,6 +20,29 @@ export function setCurrentUser( user ) {
 	};
 }
 
+export function fetchCurrentUser() {
+	return async ( dispatch ) => {
+		dispatch( {
+			type: CURRENT_USER_FETCH,
+		} );
+
+		try {
+			const user = await wpcom.me().get( {
+				meta: 'flags',
+			} );
+			const userData = filterUserObject( user );
+
+			const storedUserId = getStoredUserId();
+			if ( storedUserId != null && storedUserId !== userData.ID ) {
```

Comment:
Is `userData.ID` able to be null/undefined? Do we need both checks here or can we just check against `userData.ID`?

Reply:
Yes, it can be `null` or `undefined` by default, depending on the way we're accessing it. It's best if we check it just in case.